static/scss/detail/learning-outcomes.scss

```diff
@@ -0,0 +1,56 @@
+// sass-lint:disable mixins-before-declarations
+
+.learning-outcomes-block {
+  background: white url('#{$static-path}/images/bg-access.png') repeat-x 0 0;
+  padding: 50px 0;
+
+  .heading-container {
```

Comment:
There is an existing class applied to other elements called `header-container`. Can you change this class name to match that? Having both is confusing and would make it easy to introduce styling bugs